dataprofiler/profilers/profile_builder.py

```diff
@@ -2869,7 +2874,7 @@ def tqdm(level: set[int]) -> Generator[int, None, None]:
 
         # Generate pool and estimate datasize
         pool = None
-        if self.options.multiprocess.is_enabled:
+        if self.options.multiprocess.is_enabled and auto_multiprocess_toggle:
```

Comment:
I wonder if we can just do `if auto_multiprocess_toggle:` since that is really what turns on the multiprocess. We already use `self.options.multiprocess.is_enabled` above to change `auto_multiprocess_toggle` so I don't think it's needed here and below